app/assets/javascripts/admin/views/admin_pages_view.js

```diff
@@ -0,0 +1,44 @@
+/*global Mousetrap:true */
+
+/**
+  A view to handle site pages
+
+  @class AdminPagesView
+  @extends Discourse.View
+  @namespace Discourse
+  @module Discourse
+**/
+Discourse.AdminPagesView = Discourse.View.extend({
+  templateName: 'admin/templates/pages',
+  classNames: ['pages'],
+
+  init: function() {
```

Comment:
Instead of overriding the `init`, `didInsertElement` and `willDestroyElement` functions, it's better to "*subscribe*" to those events using `_setup: function() { ... }.on("init")` for example.